Project.toml

```
@@ -11,8 +11,8 @@ Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 
 [compat]
-ARFFFiles = "1.3"
-HTTP = "0.8, 0.9"
+ARFFFiles = "^1.4.1"
```

In Julia project files the caret `^` is understood. 

```suggestion
ARFFFiles = "1.4.1"
```